functions/src/modules/SE.functions.ts

```diff
@@ -193,81 +193,85 @@ export const createTaskFromChunks = functions.database.ref(
  * Sends a notification email to the coordinator & udpates the corresponding Task
  */
 const seUploadsBucketUrl = functions.config().sound_editing.uploads.bucket_name;
-export const uploadProcessing = functions.storage.bucket(seUploadsBucketUrl).object()
+export const processUploadedFile = functions.storage.bucket(seUploadsBucketUrl).object()
 .onFinalize(async object => {
 
   const filePath = object.name;
   const seUploadsBucket = admin.storage().bucket(seUploadsBucketUrl);
+  let uploadCode, list, taskId, task, taskRef;
+
+  if (!filePath.startsWith('restored'))
```

Comment:
Have you tested that it works? Doesn't the path start with the `/`?

Reply:
Aha, I logged the name multiple times, the file name (containing the whole path) has no slash preceding it.